javasphinx/apidoc.py

```diff
@@ -192,8 +200,14 @@ def generate_from_source_file(doc_compiler, source_file, cache_dir):
     except Exception:
         util.unexpected('Unexpected exception while parsing %s', source_file)
 
+    documents = {}
     try:
-        documents = doc_compiler.compile(ast)
+        if source_file.endswith("package-info.java"):
+            for _, node in ast.filter(javalang.tree.PackageDeclaration):
```

Comment:
I removed the filter, but it is yet required to use a loop to check where is the PackageDeclaration object inside the ast dict, because in this case it will have 2 entries where the keys are not a constant value. Since there is no order guarantee for a dict, the PackageDeclaration sometimes appears as the first entry, other times it appears as the second one.